example_sharedkey_test.go

```diff
@@ -190,7 +189,6 @@ func Example() {
 	// If the message is successfully decrypted, then client and server
 	// can talk to each other using the key they derived
 	fmt.Printf("S -> C: %s\n", message)
-	// Output: Hi client! Will you be my Valentine?
```

Comment:
only 1 `Output:` directive can be made per Example. The one at the end of this example covers all output already